pkg/config/setup/config.go

```diff
@@ -1855,16 +1871,14 @@ func LoadDatadogCustom(config pkgconfigmodel.Config, origin string, secretResolv
 }
 
 // LoadCustom reads config into the provided config object
-func LoadCustom(config pkgconfigmodel.Config, origin string, secretResolver optional.Option[secrets.Component], additionalKnownEnvVars []string) (*pkgconfigmodel.Warnings, error) {
+func LoadCustom(config pkgconfigmodel.Config, additionalKnownEnvVars []string) (*pkgconfigmodel.Warnings, error) {
 	warnings := pkgconfigmodel.Warnings{}
```

Comment:
It seems that the warning variable is not used within this function and that we always return it empty. It is later use in `LoadDatadogCustom ` function to add a warning for `TraceMallocEnabledWithPy2`

What do you think if we instead create the  `warnings := pkgconfigmodel.Warnings{}` variable in `LoadDatadogCustom` where it is used and simplify the function interface for `LoadCustom` to 

```
LoadCustom(config pkgconfigmodel.Config, additionalKnownEnvVars []string) error
```

Reply:
Ah yes nice catch !

Reply:
Note for posterity: it was already unused before my changes 🙃

Reply:
> Note for posterity: it was already unused before my changes 🙃

I know 😉